README.md

```diff
@@ -23,11 +114,3 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
```

Comment:
@SurbhiJainMSFT  to confirm if removing trademark is OK

Reply:
@aosolis for visibility. Please let us know if the Trademark section has to be included. Its currently included in the base file template but we removed it as it is not a part of README file for released app templates